### DISCUSSION TITLE
是否应该双击卡片覆盖编辑器内的内容

### DISCUSSION THREAD

**Discussion by windily-cloud** [Quorafind/Obsidian-Thino#535] [General 通用]
## 支持者

> 双击其它卡片就应该直接切换，丢失与否是用户自己行为的问题。直接覆盖能提升编辑效率，少删除再双击的一步

## 不支持者

> 新建的和编辑的 使用不同编辑器/ 编辑的内容放草稿箱/最次是覆盖之前给个弹窗。@内测 消失

> 如果能随时覆盖。那风险还挺高啊。 编辑内容的重要性绝对凌驾于用户之上  @内测 物种多样性

**Comment by xiaomingyu-git** [comment]
在移动端，本身就会极易误触。编辑状态下，锁屏/解锁的瞬间，就可能导致编辑的内容消失。直接覆盖是绝对不应该的。
参考 flomo/写拉松  /memos 都没有直接覆盖用户正在输入的内容的情况

**Comment by xiaomingyu-git** [comment]
![image](https://github.com/Quorafind/Obsidian-Thino/assets/44595719/97cae21d-3b39-4b85-b1d4-da0b70db9d84)
flomo等产品，编辑的时候都不会覆盖用户之前输入的内容。
